common-lib/src/main/scala/com/gu/mediaservice/lib/elasticsearch/Mappings.scala

```diff
@@ -60,11 +60,11 @@ object Mappings {
 
   val metadataMapping = nonDynamicObj(
     "dateTaken" -> dateFormat,
-    "description" -> sStemmerAnalysedString,
-    "byline" -> standardAnalysedString,
+    "description" -> (sStemmerAnalysedString ++ copyTo("metadata.englishAnalysedCatchAll")),
+    "byline" -> (standardAnalysedString ++ copyTo("metadata.englishAnalysedCatchAll")),
```

Comment:
How are updates handled? That is, when `byline` is updated, would `copyTo` blindly append to `englishAnalysedCatchAll` or will it replace the current byline, or recreate the entire field?